pkg/route/api/allocator/allocator.go

```diff
@@ -0,0 +1,30 @@
+package allocator
```

Comment:
I would prefer it to be presented as an interface to consumers, even if you use the static defaultAllocator method.


> On Mar 10, 2015, at 2:14 PM, ramr <notifications@github.com> wrote:
> 
> In pkg/route/api/allocator/allocator.go:
> 
> > @@ -0,0 +1,30 @@
> > +package allocator
> Ok - let me add a TODO for now. Its going to take me some time to get a handle on how to do that/how it works, so it might be safer to do it in a separate PR. 
> That sound good @smarterclayton ?
> 
> —
> Reply to this email directly or view it on GitHub.
>